distributed/reschedule.py

```diff
@@ -0,0 +1,15 @@
+from __future__ import annotations
+
+
+class Reschedule(Exception):
```

Comment:
I had to move this to avoid errors on import.

Reply:
`distributed.exceptions` is a good suggestion, I've renamed the module. `Reschedule` has been around for a while, so I'd rather not rename it as this would be a breaking change (and completely unrelated to this PR).

Reply:
👍